sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -50,8 +50,7 @@ export interface CreateRsaKeyOptions extends CreateKeyOptions {
 
 // @public
 export class CryptographyClient {
-    constructor(key: string | KeyVaultKey, // keyUrl or KeyVaultKey
-    credential: TokenCredential, pipelineOptions?: CryptographyClientOptions);
+    constructor(key: string | KeyVaultKey, credential: TokenCredential, pipelineOptions?: CryptographyClientOptions);
```

Comment:
This looks weird, but it's just an alignment after the comment was moved from this parameter to the proper place in the constructor documentation.